Ternary Search Tree/TernarySearchTree.swift

```diff
@@ -95,40 +95,40 @@ public class TernarySearchTree<Element> {
     - returns: The element, if found. Otherwise, nil.
     */
     public func find(key: String) -> Element? {
-        return findNode(root, withKey: key, atIndex: 0)
+        return find(node: root, withKey: key, atIndex: 0)
     }
 
     /**
      Helper method that performs actual legwork of find operation. Implemented recursively.
 
-     - parameter aNode:     The current node being evaluated.
+     - parameter node:      The current node being evaluated.
      - parameter key:       The key being used for the search.
      - parameter charIndex: The index of the current char in the search key
 
      - returns: The element, if found. Nil otherwise.
      */
-    private func findNode(aNode: TSTNode<Element>?, withKey key: String, atIndex charIndex: Int) -> Element? {
+    private func find(node: TSTNode<Element>?, withKey key: String, atIndex charIndex: Int) -> Element? {
 
         //Given key does not exist in tree.
-        if aNode == nil {
+        if node == nil {
```

Comment:
I think we could convert this to a guard statement and return nil if node doesn't exist.